openmapflow/pytorch_dataset.py

```diff
@@ -1,247 +1,22 @@
-from typing import Dict, List, Optional, Tuple, Union, cast
+from typing import List, Tuple
 
 import numpy as np
-import pandas as pd
 import torch
-from dateutil.relativedelta import relativedelta
 from torch import Tensor
 from torch.utils.data import Dataset
 
-from openmapflow.bbox import BBox
-from openmapflow.constants import CLASS_PROB, END, EO_DATA, LAT, LON, MONTHS, START
-from openmapflow.utils import tqdm
-
-IS_POSITIVE_CLASS = "is_positive_class"
-IS_LOCAL = "is_local"
-
-
-def _is_local(df: pd.DataFrame, bbox: Optional[BBox]) -> Union[bool, pd.Series]:
-    if bbox is None:
-        return True
-    return (
-        (df[LAT] >= bbox.min_lat)
-        & (df[LAT] <= bbox.max_lat)
-        & (df[LON] >= bbox.min_lon)
-        & (df[LON] <= bbox.max_lon)
-    )
```

Comment:
All these functions are ones the user could implement if need be, they are not absolutely necessary for a MVP OpenMapFlow project